applications/nrf_desktop/src/modules/Kconfig.dfu_mcumgr

```diff
@@ -9,20 +9,16 @@ menuconfig DESKTOP_DFU_MCUMGR_ENABLE
 	select EXPERIMENTAL
 	select CAF_BLE_SMP_TRANSFER_EVENTS if MCUMGR_TRANSPORT_BT
 	select MCUMGR
-	select MCUMGR_GRP_IMG
 	select MCUMGR_GRP_OS
```

Comment:
Do we need `CONFIG_MCUMGR_GRP_OS` for SUIT?

Reply:
Yes because:

# Allow a DFU host to retrieve information about the number and size of MCUmgr buffers.
# The nRF Connect Device Manager uses this information to speed up the DFU.
CONFIG_MCUMGR_GRP_OS_MCUMGR_PARAMS=y

Reply:
If it's only speed improvement (not an actual requirement) it should be enabled in `*.conf` file I think (or eventually it can be implied). Keep in mind that OS mgmt is required for SMP DFU on nRF52, because it enables system reboot command that is used right after finalized DFU image transfer.